### PR TITLE
Todo List App: Implement Interactive Feature

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done
- [x] Implemented interactive features
- [x] Checked linter errors.

### CODE REVIEW

File: modules/fillActivities.js [hunk 1]
```diff
@@ -10,17 +10,26 @@ const fillActivities = (activities) => {
...
+      <p class = "d${activities[a].index} activity-details">${activities[a].description}</p>
+      </div>
+      <div class = "d${activities[a].index} display-periods period-container flex-column"><span class = "period"></span><span class = "period"></span><span class = "period"></span></div>
+      <i class="d${activities[a].index}  fa-regular fa-pen-to-square modify"></i>
+      <i id = "d${activities[a].index}" class="fa-regular fa-trash-can bin"></i>
+      `;
+      activitiesArea.appendChild(activityContainer[a]);
+      document.querySelector(`.d${activities[a].index}`).style.textDecoration = 'line-through';
     }
     document.querySelector(`.d${activities[a].index}`).contentEditable = true;
```
Comment (tugsem) [inline modules/fillActivities.js]: - [ ] According to the project requirements, checked checkboxes should remain on the page upon refresh. However, when I checked an item and refreshed the page, I can't see the checkbox as it is. But the update for 'completed' immediately updates upon change when I check the storage. So I believe you can fix this easily. 🎯 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @rmiyoyo  👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️checkbox are working and updating the state
✔️new modules created 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work After Review
- [x] Fixed completed tasks logic.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @rmiyoyo 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Checkboxes are implemented as expected
-  ✅ Great work using the right GitHub Flow 
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**